src/Chapter1.hs

```diff
@@ -619,8 +633,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let lastDigit = mod n 10
+      secLastDigit = mod (div n 10) 10
```

Comment:
Great one!
You can see that this `lastDigit` here is actually the `lastDigit` function from the previous task.
Also, the negative case in here as well 🙂 

Also, I notice that you use

```haskell
mod x 10
```
and 
```haskell
div x 10
```

One hint to make your solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(a, b) = divMod x 10
```

You can see how we could pattern match on the pair 🙂